src/Objects/_PVector.js

```diff
@@ -0,0 +1,283 @@
+  };
+
+  _PVector.random2D = function(target, parent) {
+    //const rnd = parent? parent : target instanceof Processing? target : Math;
+    const rnd = parent? parent : target && target.random? target : Math;
+    return _PVector.fromAngle(TAU * rnd.random(), target);
+  };
+
+  _PVector.random3D = function(target, parent) {
+    //const rnd = parent? parent : target instanceof Processing? target : Math,
+    const rnd = parent? parent : target && target.random? target : Math,
+          ang = TAU * rnd.random(),
+          vz  = 2 * rnd.random() - 1,
+          vzr = Math.sqrt(1 - vz*vz),
+          vx  = vzr * Math.cos(ang),
+          vy  = vzr * Math.sin(ang);
+    return target? target.set(vx, vy, vz) : new _PVector(vx, vy, vz);
+  };
+
+  _PVector.dist = function(v1, v2) {
+    return Math.sqrt(_PVector.distSq(v1, v2));
+  };
+
+  _PVector.distSq = function(v1, v2) {
+    return sq(v1.x - v2.x) + sq(v1.y - v2.y) + sq(v1.z - v2.z);
+  };
+
+  _PVector.dot = function(v1, v2) {
+    return v1.x*v2.x + v1.y*v2.y + v1.z*v2.z;
+  };
+
+  _PVector.cross = function(v1, v2, target) {
+    const cx = v1.y*v2.z - v2.y*v1.z,
+          cy = v1.z*v2.x - v2.z*v1.x,
+          cz = v1.x*v2.y - v2.x*v1.y;
+    return target? target.set(cx, cy, cz) : new _PVector(cx, cy, cz);
+  };
+
+  _PVector.angleBetween = function(v1, v2) {
+    if (!v1.x && !v1.y && !v1.z || !v2.x && !v2.y && !v2.z) { return 0; }
+    const amt = _PVector.dot(v1, v2) / Math.sqrt(v1.magSq() * v2.magSq());
+    return amt <= -1? Math.PI : amt >= 1? 0 : Math.acos(amt);
+  };
+
+  _PVector.lerp = function(v1, v2, amt) {
+    return v1.copy().lerp(v2, amt);
+  };
+
+  _PVector.add = function(v1, v2, target) {
+    return target? target.set(v1.x + v2.x, v1.y + v2.y, v1.z + v2.z)
+                 : new _PVector(v1.x + v2.x, v1.y + v2.y, v1.z + v2.z);
+  };
+
+  _PVector.sub = function(v1, v2, target) {
+    return target? target.set(v1.x - v2.x, v1.y - v2.y, v1.z - v2.z)
+                 : new _PVector(v1.x - v2.x, v1.y - v2.y, v1.z - v2.z);
+  };
+
+  _PVector.mult = function(v, n, target) {
+    if (typeof n === 'number') {
+      return target? target.set(v.x*n, v.y*n, v.z*n)
+                   : new _PVector(v.x*n, v.y*n, v.z*n);
+    }
+    return target? target.set(v.x*n.x, v.y*n.y, v.z*n.z)
+                 : new _PVector(v.x*n.x, v.y*n.y, v.z*n.z);
+  };
+
+  _PVector.div = function(v, n, target) {
+    if (typeof n === 'number') {
+      return target? target.set(v.x/n, v.y/n, v.z/n)
+                   : new _PVector(v.x/n, v.y/n, v.z/n);
+    }
+    return target? target.set(v.x/n.x, v.y/n.y, v.z/n.z)
+                 : new _PVector(v.x/n.x, v.y/n.y, v.z/n.z);
+  };
+
+  _PVector.prototype = {
+    copy: function() {
+      return new _PVector(this.x, this.y, this.z);
+    },
+    get: function(target) {
+      if (!arguments.length) { return this.copy(); } // @Deprecated
+      if (typeof target !== 'object') { return this.array(); }
```

Comment:
I know, that's what I said. I said "there is no deprecation on `get()`", which as is pretty obvious is the parameter-less function call.

Anyway, can you point to where that information is listed on the Processing website or the processing github issue tracker? Because I do *not* see a deprecation notice for `get()` (again, the one *without* parameters) in the online documentation, and so if it's not there, it doesn't get mirrored in Processing.js

If that's a problem with the Processing documentation being incomplete, then I must ask you to file an issue with processing to have the documentation clarified or updated, based on your knowledge of how it's supposed to work, and then you can point to that issue as support for how the function's supposed to behave.

Right now there is no deprecation of `get()`, without arguments, according to https://processing.org/reference/PVector_get_.html, and no explanation, nor example, of what `target` is for.